config/ccms/attribute_block_configs/non_passported.yml

```diff
@@ -1597,12 +1597,6 @@ other_capital:
     response_type: boolean
     user_defined: true
     generate_block?: true
-  OTHCAPITAL_INPUT_C_17WP2_14A:
-    value: '0'
-    br100_meaning: 'Other capital: The value'
-    user_defined: true
-    response_type: currency
-    generate_block?: true
```

Comment:
It might be better to set `generate_block?: false` instead of deleting this completely. That way the attribute won't be included in the payload, but if we want to enable it later we don't have to recreate this, we can just re-enable it.